### ISSUE TITLE
`oasis uninstall`

### ISSUE THREAD

**Issue by sharmaster96** [oasislabs/oasis-cli#116]
I think it would be a useful addition to have an `oasis uninstall` CLI command.

## Details

I have experienced things not working as expected from time to time since our CLI command intrinsics have changed to varying degrees over time, and old artifacts conflict with newer ones.

I expect this is something that will be less and less of a problem moving forward as changes decrease, but I still think it would improve developer experience significantly to have an `oasis uninstall` command that lets users have a clean start.

## Acceptance Criteria
`oasis uninstall`, after confirmation message, removes all traces of Oasis from the user's system.

**Comment by ssavvides** [comment]
I agree we should have an uninstall process but I am not sure it should be part of the CLI. Also: https://github.com/oasislabs/toolstate/issues/14
